File: KoRadio/KoRadio.Model/Worker.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace KoRadio.Model
{
	public class Worker
	{
		public int WorkerId { get; set; }

		public int UserId { get; set; }

		public string? Bio { get; set; }

		public decimal? Rating { get; set; }
		public User User { get; set; }

	}
}

File: KoRadio/UI/ko_radio_desktop/lib/models/worker.dart
import 'package:json_annotation/json_annotation.dart';
import 'package:ko_radio_desktop/models/user.dart';


part 'worker.g.dart';
@JsonSerializable()
class Worker{
  int? workerId;
  int? userId;
  String? bio;
  User? user;

   Worker({required this.workerId, required this.userId, this.bio,this.user});

  factory Worker.fromJson(Map<String, dynamic> json) => _$WorkerFromJson(json);

  Map<String, dynamic> toJson() => _$WorkerToJson(this);
}